routes/project.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
 /**
  * Get a list of projects.
  * @name get-projects
+ * @param {Object} query - The request URL query parameters
+ * @param {string} query.name - Name of project to filter by (optional)
```

Comment:
You can actually connote optional parameters with square brackets; for example:

https://github.com/osmlab/to-fix-backend/blob/10e3e31ae2a72f8928a559b9bc6495d5a804e4a2/routes/tag.js#L32-L33

documentation.js will automatically add "optional" to the documentation.